src/main/java/de/qabel/desktop/ui/accounting/GsonIdentity.java

```diff
@@ -0,0 +1,43 @@
+package de.qabel.desktop.ui.accounting;
+
+import com.google.gson.annotations.Expose;
+import java.util.HashMap;
+
+
+public class GsonIdentity extends GsonEntity {
+
+	@Expose
+	private HashMap<String, byte[]> keys = new HashMap<>();
```

Comment:
hint: it's more flexible to use the Interface `java.util.Map` instead of the concrete implementation.